projects/angular-calendar/schematics/ng-add/version-names.ts

```diff
@@ -0,0 +1,3 @@
+export const angularCalendarVersion = '0.27.19';
+export const momentVersion = '2.24.0';
+export const dateFnsVersion = '1.30.1';
```

Comment:
Let's pull these versions from the root package.json instead. Add this command to the root package.json `"pretest:schematics": "npm run build:copy-package-json",` (so the tests work with this change).

and then this file can become:
```
const packageJson = require('../../package.json'); // tslint:disable-line no-var-requires

export const angularCalendarVersion = `^${packageJson.version}`;
export const momentVersion = packageJson.devDependencies.moment;
export const dateFnsVersion = packageJson.devDependencies['date-fns'];

```